backend/mora/service/employee.py

```diff
@@ -35,6 +35,200 @@
                             url_prefix='/service')
 
 
+class BrugerRequestHandler(handlers.RequestHandler):
+    __slots__ = ['details_requests']
```

Comment:
I know this is a total nit, but `__slots__` is usually a tuple not a list. Although unlikely to matter in practice, tuples are unmodifiable:
```suggestion
    __slots__ = ('details_requests',)
```

Reply:
yup